Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-slim
+LABEL maintainer="testtest@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes if unnecessary files are included. Ensure your `.dockerignore` file is properly configured to exclude files that are not needed in the image.